test/services/fileStorage/aws/awsMultipleS3.test.js

```diff
@@ -0,0 +1,118 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const { expect } = require('chai');
+const mockery = require('mockery');
+const { Configuration } = require('@schul-cloud/commons');
+const mockAws = require('./s3.mock');
+const logger = require('../../../../src/logger');
+const app = require('../../../../src/app');
+
+const testObjects = require('../../helpers/testObjects')(app);
+
+chai.use(chaiHttp);
+
+describe('multple S3 AWS file storage strategy', () => {
+	let aws;
+
+	const options = {
+		schoolId: '0000d186816abba584714c5f',
+	};
+
+	const ShouldFail = new Error('It succeeded but should have returned an error.');
+
+	let configBefore = {};
+
+	before(async () => {
+		// Enable mockery to mock objects
```

Comment:
Note für mich noch klären: role static services Initialisierung vs Permissions die vorhanden sind?

Reply:
Teste ich in dem role static pr sobald das gemerged wurde.